src/app/website/src/careers.ml

```diff
@@ -0,0 +1,225 @@
+               digital&nbsp;lives."
+          ; Qualities.(
+              create
+                [ Quality.create "Open Source"
+                    [ text
+                        "We passionately believe in the open-source \
+                         philosophy, and make our software free for the \
+                         entire world to&nbsp;use."
+                    ; a
+                        [href "/static/code.html"; Style.render linkstyle]
+                        [text "Take&nbsp;a&nbsp;look&nbsp;→"] ]
+                ; Quality.create "Collaboration"
+                    [ text
+                        "The problems we face are novel and challenging and \
+                         we take them on as a&nbsp;team." ]
+                ; Quality.create "Inclusion"
+                    [ text
+                        "We’re working on technologies with the potential \
+                         to reimagine social structures. We believe it’s \
+                         important to incorporate diverse perspectives from \
+                         conception through&nbsp;realization." ] ])
+          ; Benefits.(
+              create
+                [ Benefit.create "Healthcare" ~top_style:""
+                    [ [ text
+                          "We cover 100% of employee premiums for platinum \
+                           healthcare plans with zero deductible, and 99% of \
+                           vision and dental&nbsp;premiums" ] ]
+                ; Benefit.create "401k"
+                    [ [ text
+                          "401k contribution matching up to 3% of&nbsp;salary"
+                      ] ]
+                ; Benefit.create "Education"
+                    [ [ text
+                          "$750 annual budget for conferences of your choice \
+                           (we cover company-related&nbsp;conferences)" ]
+                    ; [text "Office library"]
+                    ; [text "Twice-a-week learning lunches"] ]
+                ; Benefit.create "Equipment"
+                    [ [ text
+                          "Top-of-the-line laptop, $500 monitor budget and \
+                           $500 peripheral budget" ] ]
+                ; Benefit.create "Time&nbsp;off"
+                    [ [ text
+                          "Unlimited vacation, with encouragement for \
+                           employees to take off"
+                      ; span [Style.just "i"] [text "at least"]
+                      ; text "14 days&nbsp;annually" ] ]
+                ; Benefit.create "Meals"
+                    [[text "Healthy snacks and provided lunch twice a week"]]
+                ; Benefit.create "Other"
+                    [ [text "Parental leave"]
+                    ; [text "Commuting benefits"]
+                    ; [text "Bike-friendly culture"]
+                    ; [ text
+                          "Take up to 1 day (8 hours) of paid time off per \
```

Comment:
The volunteer text does not fit on one line and it actually goes to the next line. I thought the next line was another different benefit but it wasn't. Maybe we should have bullets to distinguish what is a benefit and what is a continuation of the next statement. 

The easiest thing to do is probably to make the statement shorter.